pkg/blockbuilder/tsdb.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 type builder interface {
-	process(ctx context.Context, rec *kgo.Record, blockMin, blockMax int64, recordProcessedBefore bool) (_ bool, err error)
+	process(ctx context.Context, rec *kgo.Record, lastBlockMax, blockMax int64, recordProcessedBefore bool) (_ bool, err error)
```

Comment:
I changed the name because technically it is not a blockMin, rather the max of the last cycle